apps/frontend/src/store/search_filter_sync.ts

```diff
@@ -18,6 +18,7 @@ export type CciSearchTerm = string;
 export type IaControlsSearchTerm = string;
 export type NistIdFilter = string;
 export type KeywordsSearchTerm = string;
+export type FilenameSearchTerm = string;
```

Comment:
yeah i guess we can ignore the sonarqube finding here since it does get used but in a different file